tox.ini

```diff
@@ -23,12 +23,19 @@ commands =
     twine check dist/*
 
 [testenv:release]
-basepython = python3
+skip_install = True
 deps =
-    wheel
+    pep517>=0.5
+    twine>=1.13
+    path.py
+passenv =
+    TWINE_PASSWORD
+setenv =
+    TWINE_USERNAME = {env:TWINE_USERNAME:__token__}
 commands =
-    python setup.py -q bdist_wheel sdist
-    twine upload --skip-existing dist/*
+    python -c "import path; path.Path('dist').rmtree_p()"
```

Comment:
File this under "today I learned". Neat.

Reply:
Cool. Yeah, so much better than `rm -r`, which only works on some platforms.

Reply:
Why not more simply:

```
python -c 'import os; os.removedirs("dist")'
```

https://docs.python.org/3/library/os.html#os.removedirs

Reply:
I don't think `os.removedirs` does what `rmtree_p` does. By my estimation (untested):

- `rmtree_p` removes a directory and its decendants and doesn't error if the directory doesn't exist.
- `removedirs` removes an empty directory and any empty ancestors mentioned in the path and errors if the directory doesn't exist.